end-to-end-test/remote/specs/core/oncoprint.screenshot.spec.js

```diff
@@ -222,8 +228,11 @@ describe('track group headers', function() {
     it('oncoprint should return to non-clustered state correctly', function() {
         // Cluster the mrna heatmap group
         var mrnaElements = getGroupHeaderOptionsElements(2);
-        browser.click(mrnaElements.button_selector);
-        browser.waitForVisible(mrnaElements.dropdown_selector, 1000); // wait for menu to appear
+        setDropdownOpen(
+            true,
+            mrnaElements.button_selector,
+            mrnaElements.dropdown_selector
+        );
```

Comment:
same for this one: https://circleci.com/gh/cBioPortal/cbioportal-frontend/62692?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link